### PR TITLE
476 gov shutdown

### PR DESCRIPTION
**Pull request by mtlaney**
## Summary
Addresses Issue # 
#476 
Please note if fully resolves the issue per the acceptance criteria: Y

*Describe the pull request here, including any supplemental information needed to understand it.*
Add a shutdown build trigger, which replaces the router file with a wildcard shutdown component. Every page attempted to visit will display the shutdown message.

ng serve -c shutdown to run tests.

## Impacted Areas of the Site
All router links will be inaccessible during the shutdown build.

## Optional Screenshots
screen shot 2018-12-19 at 2 15 09 pm

## This pull request changes...
- [ ] add optional build trigger for shutdown in the event of a December 21st shutdown.

## This pull request is ready to merge when...
- [ ] Feature branch is starts with the issue number
- [ ] Is connected to its original issue via zenhub 👇
- [ ] Tests have been updated 
- [ ] All tests are passing and meet coverage, linting, and accessibility requirements. And no security vulnerabilities ⚫️(Circle)
- [ ] Server actions captured by logs (manual)
- [ ] Documentation / readme.md updated (manual)
- [ ] API docs updated if need (manual)
- [ ] JSDocs updated (manual)
- [ ] Docker updated if needed (manual)
- [ ] This code has been reviewed by someone other than the original author

### CODE REVIEW

**Comment by mtlaney** [comment]
@lauraGgit 👍

**Comment by mtlaney** [comment]
@lauraGgit Added new commit. Removed unneeded routes, reverted tempoutfitters, and updated package.json, could use either your or @davemcorwin help on the circle config file, admittedly I am new to circle.

**Comment by lauraGgit** [comment]
@mtlaney looks good, for circle if you just update https://github.com/mtlaney/fs-open-forest-platform/blob/476_gov_shutdown/.circleci/config.yml#L278 to your `npm run shutdown`.

And `https://github.com/mtlaney/fs-open-forest-platform/blob/476_gov_shutdown/.circleci/config.yml#L275` to `Create frontend compiled distribution for shutdown`. 

You'll be all set!

**Comment by lauraGgit** [comment]
For context, this is in our `prod_deploy` block of code that only fires when we merge to master. It will compile the build for the frontend and then run a bash script which has the cloud foundry cli (what cloud.gov is based on) to push our application to production, using zero-downtime deployment.

**Comment by mtlaney** [comment]
@lauraGgit added commit for circle yaml changes

**Comment by mtlaney** [comment]
running unit / e2e / lint quick, than will push the /shutdown link to regular routes as well

File: frontend/angular.json [hunk 1]
```diff
@@ -52,6 +52,18 @@
                 }
               ]
             },
+            "shutdown": {
+              "fileReplacements": [
```
Comment (davemcorwin) [inline frontend/angular.json]: should this also have `"aot": false` like in the production config?

**Pull request merged**